### PR TITLE
Add parameter to manage service

### PR DESCRIPTION
**Pull request by chr1s692**

### CODE REVIEW

File: manifests/service.pp [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # @summary Set up the foreman service
 # @api private
 class foreman_proxy::service {
-  service { 'foreman-proxy':
-    ensure => running,
-    enable => true,
+  if $foreman_proxy::manage_service == true
```
Comment (ekohl) [inline manifests/service.pp:4]:
```suggestion
  if $foreman_proxy::manage_service {
```

File: manifests/init.pp [hunk 2]
```diff
@@ -7,6 +7,9 @@
 #
 # $ensure_packages_version::    control extra packages version, it's passed to ensure parameter of package resource
 #
+# $manage_service:                     control the service, wether it should be started / enabled or not. usefull, if the
```
Comment (ekohl) [inline manifests/init.pp:10]: This should be an advanced parameter, under the `Advanced Parameters`

**Review by ekohl — CHANGES_REQUESTED** [review]

File: manifests/init.pp (hunk 2 above)
Comment (chr1s692) [reply manifests/init.pp]: hi @ekohl ,
moved to Advanced Parameters :)

**Review by ekohl — APPROVED** [review]

File: manifests/init.pp [hunk 3]
```diff
@@ -279,9 +279,13 @@
 #
 # $registration_url::           URL that hosts will connect to when registering
 #
+# $manage_service:              control the service, whether it should be started / enabled or not. useful, if the
```
Comment (ekohl) [inline manifests/init.pp:282]:
```suggestion
# $manage_service::             control the service, whether it should be started / enabled or not. useful, if the
```

**Pull request merged**